pkg/ctl/update/cluster.go

```diff
@@ -0,0 +1,99 @@
+package update
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha4"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/printers"
+)
+
+var updateClusterDryRun = true
+
+func updateClusterCmd(g *cmdutils.Grouping) *cobra.Command {
+	p := &api.ProviderConfig{}
+	cfg := api.NewClusterConfig()
+
+	cmd := &cobra.Command{
+		Use:   "cluster",
+		Short: "Update cluster",
+		Run: func(_ *cobra.Command, args []string) {
+			if err := doUpdateClusterCmd(p, cfg, cmdutils.GetNameArg(args)); err != nil {
+				logger.Critical("%s\n", err.Error())
+				os.Exit(1)
+			}
+		},
+	}
+
+	group := g.New(cmd)
+
+	group.InFlagSet("General", func(fs *pflag.FlagSet) {
+		fs.StringVarP(&cfg.Metadata.Name, "name", "n", "", "EKS cluster name (required)")
+		cmdutils.AddRegionFlag(fs, p)
+		cmdutils.AddVersionFlag(fs, cfg.Metadata, "")
+		fs.BoolVar(&updateClusterDryRun, "dry-run", updateClusterDryRun, "do not apply any change, only show what resources would be added")
+	})
+
+	cmdutils.AddCommonFlagsForAWS(group, p, false)
+
+	group.AddTo(cmd)
+	return cmd
+}
+
+func doUpdateClusterCmd(p *api.ProviderConfig, cfg *api.ClusterConfig, nameArg string) error {
```

Comment:
Can we print the Kubernetes version and ask for confirmation? 

Something like:
```
The gopher cluster will be upgrade to 1.13.3 do you want to proceed? yest/no
```

Reply:
We have `--dry-run` on by default. This doesn't actually do version updates, that's for another PR.